pkg/operator/defaults.go

```diff
@@ -66,8 +66,6 @@ var (
 		"v2.51.1",
 		"v2.51.2",
 		"v2.52.0",
```

Comment:
can we keep the comment for future ourselves? The issue is that v2.52.1 is listed in the Prometheus CHANGELOG (we could file an issue to Prometheus but I suppose that it's too late).
```suggestion
		"v2.52.0",
		// The v2.52.1 image tag is missing from docker.io and quay.io registries.
```